packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Providers/MrwSerializationTypeDefinitions/TestData/SerializationCustomizationTests/CanCustomizeExtensibleEnum(string).cs

```diff
@@ -0,0 +1,145 @@
+// <auto-generated/>
+
+#nullable disable
```

Comment:
any way to narrow down the important parts you are trying to test here?  emitting the entire file increases the changes of diffs / test updates when we change minor things in the serialization file such as order of methods emitted etc.

Reply:
https://github.com/microsoft/typespec/issues/4729